libraries/botframework-connector/src/teams/models/index.ts

```diff
@@ -128,7 +128,7 @@ export type TeamsMeetingInfoResponse = TeamsMeetingInfo & {
 /**
  * Contains response data for the sendMeetingNotification operation.
  */
-export type TeamsSendMeetingNotificationResponse = TeamsMeetingNotificationRecipientFailureInfos & {
+export type TeamsMeetingNotificationResponse = MeetingNotificationResponse & {
```

Comment:
Is there any reason why we didn't remove `Teams `for `TeamsMeetingNotificationResponse`

Reply:
So the reason being is that in this file, all the Teams related models are prefixed with `Teams`. Plus, this is simply an implementation detail and not being exposed to the end user, so there is no inconsistency to worry about.